<launch>
     <arg name="world_file" default="$(find multiple_robots_stage)/maps/worlds/twoRobotsCropped.world" />
     <arg name="map_file" default="$(find multiple_robots_stage)/maps/twoRoomsCropped.yaml" />
     <arg name="waypoints" default="0" /><arg name="robotName1" default="robot_0" />
     <arg name="robotName2" default="robot_1" />
     <arg name="robotName3" default="robot_2" />
     <master auto="start" />
     <param name="/use_sim_time" value="true" />
     <node pkg="map_server" type="map_server" name="map_server" args="$(arg map_file)" respawn="false">
     <param name="frame_id" value="/map" /></node>
     <node pkg="stage_ros" type="stageros" name="stageros" args="$(arg world_file)" respawn="false">
     <param name="base_watchdog_timeout" value="0.2" />
     </node><group ns="robot_0">
         <param name="tf_prefix" value="robot_0"/> 
<include file="$(find multiple_robots_stage)/launch/client_robot.launch">
             <arg name="robot_name" value="robot_0" />
         </include>
<include             file="$(find multiple_robots_stage)/launch/includes/single_robot.launch">
             <arg name="namespace" value="robot_0" />
             <arg name="initial_pose_x" value="2.0" />
             <arg name="initial_pose_y" value="9.0" />
             <arg name="initial_pose_a" value="0.0" />
             <arg name="global_planner" value="ant_global_planner/GlobalPlanner" />
         </include>
</group><group ns="robot_1">
         <param name="tf_prefix" value="robot_1"/> 
<include file="$(find multiple_robots_stage)/launch/client_robot.launch">
             <arg name="robot_name" value="robot_1" />
         </include>
<include             file="$(find multiple_robots_stage)/launch/includes/single_robot.launch">
             <arg name="namespace" value="robot_1" />
             <arg name="initial_pose_x" value="3.0" />
             <arg name="initial_pose_y" value="10.0" />
             <arg name="initial_pose_a" value="0.0" />
             <arg name="global_planner" value="ant_global_planner/GlobalPlanner" />
         </include>
</group></launch>